app/views/katello/api/v2/flatpak_remote_repositories/show.json.rabl

```diff
@@ -0,0 +1,13 @@
+object @resource
+
+extends "katello/api/v2/flatpak_remote_repositories/base"
+
+child :flatpak_remote => :flatpak_remote do
+  attributes :id, :name, :url
+end
+
+if params[:manifests]
```

Comment:
I think we may need a boolean conversion here, since the following returns manifests:
```
$ curl https://`hostname`/katello/api/v2/flatpak_remote_repositories/28?manifests=false
```